docs/guide/custom-ui.md

```diff
@@ -232,10 +234,11 @@ Do not use core actions as your `action` property value - this would lead to unp
 :::
 
 ::: note
-* The current API supports only `context: "update"`.
+* The current API supports only `context: "update"` (the custom menu items are available for previously saved documents).
 * The `action` property should be globally unique.
 * Overriding the same `action` is possible (the last wins).
 * You may mark the action as "dangerous" via an optional property `modifiers: [ "danger" ]`.
+* An additional optional boolean property `manuallyPublished` is supported. When set to `true`, the custom menu will be shown only for document types with options `autopublish: false` and `localized: true`.
```

Comment:
Rephrase: "When set to `true`, the custom menu item is available only for document types that do not have the `autopublish: true` or `localized: false` options set." This avoids the implication that you have to explicitly set them the other way around (you don't, because the defaults are compatible with custom menu items).

Also "menu item" not "menu" (sorry to be a pain).